Source/bmalloc/bmalloc/VMAllocate.h

```diff
@@ -139,60 +180,39 @@ inline void* vmAllocate(size_t vmSize, VMTag usage = VMTag::Malloc)
 inline void vmDeallocate(void* p, size_t vmSize)
 {
     vmValidate(p, vmSize);
+#if BPLATFORM(WIN)
+    auto success = VirtualFree(p, vmSize, MEM_DECOMMIT);
+    RELEASE_BASSERT(success);
+#else
     munmap(p, vmSize);
+#endif
 }
 
 inline void vmRevokePermissions(void* p, size_t vmSize)
 {
     vmValidate(p, vmSize);
+#if BPLATFORM(WIN)
+    auto success = VirtualProtect(p, vmSize, PAGE_NOACCESS, nullptr);
```

Comment:
Is it correct? I think `VirtualProtect` is not corresponding to `mprotect`. See `OSAllocator::protect` implementation.